datadog/resource_datadog_integration_gcp.go

```diff
@@ -108,35 +111,22 @@ func resourceDatadogIntegrationGcpCreate(ctx context.Context, d *schema.Resource
 		return utils.TranslateClientErrorDiag(err, httpresp, "error creating GCP integration")
 	}
 
-	d.SetId(projectID)
+	d.SetId(fmt.Sprintf("%s:%s", projectID, clientEmail))
 
-	return resourceDatadogIntegrationGcpRead(ctx, d, meta)
+	return updateDatadogIntegrationGCPState(d, meta)
```

Comment:
```suggestion
	return resourceDatadogIntegrationGcpRead(ctx, d, meta)
```
Am I missing something?

Reply:
Why do you think we need the context?  We don't really use it so its an unnecessary extra argument for now